src/index.html

```diff
@@ -16,17 +16,29 @@
       href="style.css"
     />
   </head>
-  <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
+  <body class="search-bar__container">
+    <form
+      class="search-bar search-bar--top"
+      data-qa="big"
+    >
+      <input
+        class="search-bar__input"
+        type="text"
+        data-qa="keypress"
+        placeholder="Try “Los Angeles“"
+      />
```

Comment:
Ensure that the indentation between parent and child elements is consistent and uses 2 spaces. The <input> element inside the <form> should be indented with 2 spaces more than the <form>.